Code-the-Dream-School/dd-prac-team4-front
# PS-197 Album marketplace pagination

**Description:**
Added album marketplace pagination for improved user experience.

**Testing Steps:**

- Navigate to the album marketplace page [http://localhost:3000/home](http://localhost:3000/home) in the web application.

- Verify that the pagination UI is displayed at the bottom of the album list.

**Test the following scenarios:**

1. Click on different page numbers in the pagination component to switch between pages.
2. Verify that the album list updates accordingly based on the selected page.
3. Perform searches using different search terms and search types and verify that pagination remains functional.
4. Ensure that the pagination feature does not introduce any console errors or warnings.

<img width="1402" alt="Screenshot 2023-09-09 at 1 10 39 AM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/2952900/5ae08687-dbad-4092-922d-34363cc19b87">
